doc/langref.html.in

```diff
@@ -80,11 +81,35 @@
           font-weight: bold;
       }
 
+      #main-wrapper {
+        display: flex;
+        flex-direction: column;
+        height: 100vh;
+      }
+
+      #contents-wrapper {
+        flex-grow: 1;
+        padding: 0 2em;
+      }
+
       #contents {
         max-width: 60em;
         margin: auto;
       }
 
+      #index {
+        padding: 0 1em;
+      }
+
+      @media screen and (min-width: 1025px) {
```

Comment:
Shouldn't this be measured in `em` instead of `px`?

Reply:
`px` is fine, it scales with resolution.